contrib/stack/topsStack/stackSentinel.py

```diff
@@ -11,6 +11,9 @@
 import time
 import numpy as np
 
+import geopandas as gpd
+from shapely.geometry import Point, Polygon
```

Comment:
Can these imports be moved inside the `generate_geopolygon` function? These python packages aren't formal prerequisites of isce2 so I'd like to avoid ImportErrors when they aren't otherwise needed